testdata/rust/rust-stable/src/main.rs

```diff
@@ -0,0 +1,14 @@
+use ferris_says::say;
+
+use std::io::{stdout, BufWriter};
+
+fn main() {
```

Comment:
Would it be easy to change these to be closer to the go examples (check version via API, error out if the version is not the one the example expects)

Reply:
sure, I can look into doing that.